02_housing/1a_housing_place.R

```diff
@@ -58,7 +57,7 @@ library(tidylog)
 
 # Either run "0_housing_microdata.qmd" OR: Import the already prepared microdata file 
 # this one should already match the PUMAs to places
-acs2022 <- read_csv("data/temp/2022microdata.csv") 
+acs2022 <- read_csv(here::here("data","temp","2022microdata.csv")) 
```

Comment:
It looks like you updated the read_csv to use `here::here` but isn't the data in a temp folder in the root directory not the housing subdirectory? I had to edit this to get it to work. It also doesn't look like you changed this on the 2a and 2b programs so whatever we decide should make it consistent.

Reply:
I am still having some issues with this `here::here` code. It keeps trying to use a subfolder where the data doesn't exist.